Src/AutoFixture/RandomRangedNumberGenerator.cs

```diff
@@ -135,13 +133,18 @@ private static long ConvertLimit(object limit)
 
                 case TypeCode.UInt64:
                         return (long)(ulong)limit;
-
-               
             }
 
             throw new ArgumentException("Limit parameter is non-numeric ", "limit");         
-        }      
-       

```

Comment:
I removed a few unnecessary blank lines here and there, by following [the boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule).